user-interface/frontend/themes/datamanager/components/all.css

```diff
@@ -1,8 +1,332 @@
+/****************************************
+
+Data Manager custom CSS layout definition
+
+*****************************************/
+
+/*
+First some general property definitions
+*/
+:root {
+  --dialog-height-small: clamp(min-content, 50vh, 75vh);
+  --dialog-height-medium: clamp(min-content, 75vh, 100vh);
+  --dialog-height-large: 100vh;
+  --dialog-height-max-small: 500px;
+  --dialog-height-max-medium: 800px;
+  --dialog-height-max-large: 100vh;
+  --dialog-width-small: 50vw;
+  --dialog-width-medium: 75vw;
+  --dialog-width-large: 80vw;
+  --dialog-width-max-small: 600px;
+  --dialog-width-max-medium: 1000px;
+  --dialog-width-max-large: 1200px;
+  --font-weight-light: 300;
+  --font-weight-regular: 400;
+  --font-weight-medium: 500;
+  --font-weight-semi-bold: 600;
+  --font-weight-bold: 700;
+  --icon-color-info: rgba(22, 118, 243, 1);
```

Comment:
NIT: why not hexadecimal? Usually hex notation is used for colors in css and thus is supported in most tools (including intellij and firefox) displaying the color inline when editing the css. No strong oppinion just stands out somewhat.

Reply:
why does it matter? its part of the CSS 3.0 standard and it also works in IntelliJ.

Reply:
doesn`t really, thats why I started with NIT 😄 Was just unexpected on my end.